feat: Add a shutdown notice to the dialog and all main site pages.

shane-tomlinson opened the pull request:
@rfk and @relud - this is to add a Persona shutdown notice everywhere. 😢

Commit:
feat: Add a shutdown notice to the dialog and all main site pages.

relud commented:
I don't actually understand how these changes work. It looks like the notice is being moved from 2 places to 2 different places. I'm assuming that the 2 new places somehow get included everywhere, including the approximate location of the previous places?

rfk commented:
> I'm assuming that the 2 new places somehow get included everywhere, 
> including the approximate location of the previous places?

Yes.  Adding it to layout.ejs causes it to appear on all pages of the persona.org static site, including where it was before, like this:

![screen shot 2016-10-07 at 14 06 07](https://cloud.githubusercontent.com/assets/34695/19177728/7047a280-8c97-11e6-9a35-8beb88bee2e0.png)

Adding it to dialog_layout.ejs causes it to appear in the user-facing login dialog, like this:

![screen shot 2016-10-07 at 14 06 49](https://cloud.githubusercontent.com/assets/34695/19177737/8ab9bfea-8c97-11e6-90e8-459d8951410d.png)

@shane-tomlinson one think I did notice, is that clicking on the "More info" link in the dialog doesn't work for me.  Right-clicking and opening in a new tab does, but plain left-clicking does not.  I wonder if it's intercepting navigation attempts and rejecting ones it doesn't recognize, as it seems to want to open it in the originating window.

Otherwise, this LGTM, thanks!

Commit:
fix: Make sure "more info" is fully clickable from `/`.

The `#hAlign` element had a -90 margin, and since it came after
the notice in the DOM, it partially overlayed the notice and
made the `More info` link unclickable.

Remove the -90 margin, and instead use JS to
center #hAlign, using the same code that already existed.

shane-tomlinson commented:
@rfk - thanks! fixed the click issue. The odd thing was I tested that, the link was being partially obscured. If you clicked on the right side of the link everything worked as expected, from the left side, no.
